spec/vlan_list_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require 'chef/knife/vsphere_vlan_list'
+
+describe Chef::Knife::VsphereVlanList do
+
+  let(:datacenter) { double('Datacenter') }
+  let(:n1) { double('Network', name: 'n1') }
+  let(:n2) { double('Network', name: 'n2') }
+
+  it 'enumerates the vlans' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.